WalletWasabi/Extensions/NBitcoinExtensions.cs

```diff
@@ -353,13 +353,19 @@ void Walk(TransactionDependencyNode node)
 
 	public static ScriptPubKeyType? GetInputScriptPubKeyType(this PSBTInput i)
 	{
+		if (i.WitnessUtxo is not { })
+		{
+			return null;
+		}
+
```

Comment:
Nop. Alll methods need valid inputs in order to be able to operate and in case the inputs are not valid then we need to communicate that to the caller. But never never ever return garbage, never. 

This is like a vendetta: "Ohh you invoked me with garbage!? I will give you back more garbage! Here you have motherfucker" no, that's not how it works.

Remember: never receive null and never return null.